test/methods/geom_relations.jl

```diff
@@ -0,0 +1,178 @@
+# Tests of DE-9IM Methods
+
+pt1 = LG.Point([0.0, 0.0])
+pt2 = LG.Point([5.0, 5.0])
+pt3 = LG.Point([1.0, 0.0])
+pt4 = LG.Point([0.5, 0.0])
+pt5 = LG.Point([0.5, 0.25])
+pt6 = LG.Point([0.6, 0.4])
+pt7 = LG.Point([0.4, 0.8])
+
+l1 = LG.LineString([[0.0, 0.0], [1.0, 0.0], [1.0, 1.0]])
+l2 = LG.LineString([[0.0, 0.0], [1.0, 0.0]])
+l3 = LG.LineString([[-1.0, 0.0], [0.0, 0.0], [1.0, 0.0]])
+l4 = LG.LineString([[0.5, 0.0], [1.0, 0.0], [1.0, 0.5]])
+l5 = LG.LineString([[0.0, 0.0], [-1.0, -1.0]])
+l6 = LG.LineString([[2.0, 2.0], [0.0, 1.0]])
+l7 = LG.LineString([[0.5, 1.0], [0.5, -1.0]])
+l8 = LG.LineString([[0.0, 0.0], [0.5, 0.0], [0.5, 0.5], [1.0, -0.5]])
+l9 = LG.LineString([[0.0, 1.0], [0.0, -1.0], [1.0, 1.0]])
+l10 = LG.LineString([[0.0, 0.0], [1.0, 0.0], [1.0, 1.0], [0.0, 0.0]])
+l11 = LG.LineString([[0.0, 0.0], [1.0, 0.0], [1.0, 1.0], [0.0, 0.0], [-1.0, 0.0]])
+l12 = LG.LineString([[0.6, 0.5], [0.6, 0.9]])
+l13 = LG.LineString([[2.0, 2.0], [3.0, 3.0]])
+l14 = LG.LineString([[0.6, 0.25], [0.6, 0.35]])
+l15 = LG.LineString([[0.0, 3.0], [4.0, 3.0]])
+l16 = LG.LineString([[0.3, -0.7], [1.0, 0.0], [3.0, 0.6]])
+
+r1 = LG.LinearRing([[0.0, 0.0], [1.0, 0.0], [1.0, 1.0], [0.0, 0.0]])
+r2 = LG.LinearRing([[0.5, 0.2], [0.6, 0.4], [0.7, 0.2], [0.5, 0.2]])
+r3 = LG.LinearRing([[0.2, 0.7], [0.4, 0.9], [0.5, 0.6], [0.2, 0.7]])
+r4 = LG.LinearRing([[0.0, 0.0], [1.0, 0.0], [1.0, 1.0], [0.0, 1.0], [0.0, 0.0]])
+r5 = LG.LinearRing([[0.0, 0.0], [1.0, 0.0], [1.0, 1.0], [2.0, 1.0], [1.0, 2.0], [1.0, 1.0], [0.0, 0.0]])
+r6 = LG.LinearRing([[0.0, 0.0], [-1.0, 0.0], [-1.0, 1.0], [0.0, 1.0], [0.0, 0.0]])
+r7 = LG.LinearRing([[0.5, 0.5], [1.5, 0.5], [1.5, 1.5], [0.5, 1.5], [0.5, 0.5]])
+r8 = LG.LinearRing([[0.1, 0.1], [0.2, 0.1], [0.2, 0.2], [0.1, 0.2], [0.1, 0.1]])
+r9 = LG.LinearRing([[1.0, -0.5], [0.5, -1.0], [1.5, -1.0], [1.0, -0.5]])
+r10 = LG.LinearRing([[0.5, 0.2], [0.6, 0.4], [0.3, 0.3], [0.5, 0.2]])
+r11 = LG.LinearRing([[0.55, 0.21], [0.55, 0.23], [0.65, 0.23], [0.66, 0.21], [0.55, 0.21]])
+
+p1 = LG.Polygon(r4, [r2, r3])
+p2 = LG.Polygon(r4)
+p3 = LG.Polygon(r2)
+p4 = LG.Polygon(r6)
+p5 = LG.Polygon(r9)
+p6 = LG.Polygon(r11)
+p7 = LG.Polygon(r7)
+p8 = LG.Polygon([[[0.0, 0.0], [0.0, 3.0], [1.0, 2.0], [2.0, 3.0], [3.0, 2.0], [4.0, 3.0], [4.0, 0.0], [0.0, 0.0]]])
+p9 = LG.Polygon([[[0.0, 0.0], [0.0, 3.0], [1.0, 4.0], [2.0, 3.0], [3.0, 4.0], [4.0, 3.0], [4.0, 0.0], [0.0, 0.0]]])
+p10 = LG.Polygon([
+    [[0.1, 0.5], [0.1, 0.99], [0.6, 0.99], [0.6, 0.5], [0.1, 0.5]],
+    [[0.15, 0.55], [0.15, 0.95], [0.55, 0.95], [0.55, 0.55], [0.15, 0.55]]
+])
+p11 = LG.Polygon(r3)
+
+mpt1 = LG.MultiPoint([pt1, pt2])
+mpt2 = LG.MultiPoint([pt2, pt3])
+mpt3 = LG.MultiPoint([pt4, pt5])
+ml1 = LG.MultiLineString([l5, l6, l7])
+ml2 = LG.MultiLineString([l1])
+mp1 = LG.MultiPolygon([p1])
+mp2 = LG.MultiPolygon([p6, p7])
+gc1 = LG.GeometryCollection([pt1, l5, p6])
+
+test_pairs = [
+    # Points and geometries
+    (pt1, pt1, "pt1", "pt1"),  # Same point
```

Comment:
Can we put these descriptions like "Same point" in thd tuple and print it with the other info?